### PR TITLE
Add test for Rand

### PR DESCRIPTION
**Pull request by t1m-l**

### CODE REVIEW

**Comment by t1m-l** [comment]
The test is wrong, as I forgot to use the property

File: test/RandTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        return false;
+      }
+    }
+
+    return true;
+  }
+
+  public static void main(String... args) {
+    for (int i = 0; i < property_ContainsOnlyUpperCaseLatinLetters_NUM_RUNS; i++) {
+      Rand.make(testStringLen);
```
Comment (yegor256) [inline test/RandTest.java]: @Sitiritis you forgot to use the method `property_ContainsOnlyUpperCaseLatinLetters`?

**Comment by yegor256** [comment]
@Sitiritis there is a build failure, pls take a look

**Comment by yegor256** [comment]
@Sitiritis still broken

**Comment by t1m-l** [comment]
The test won't pass until pull request #18 is merged.

**Pull request closed without merging**